clang/test/Analysis/stream-error.c

```diff
@@ -303,3 +346,29 @@ void error_indeterminate_feof2(void) {
   }
   fclose(F);
 }
+
+void error_indeterminate_feof3(void) {
```

Comment:
This test seems to be not necessary (previous tests cover these conditions), otherwise a more meaningful name should be chosen for it.